var/spack/repos/builtin/packages/meson/package.py

```diff
@@ -89,6 +61,10 @@ class Meson(PythonPackage):
     # Python 3.12 detection support
     patch("python-3.12-support.patch", when="@1.1:1.2.2")
 
+    # Broken config.h generation
+    # https://github.com/mesonbuild/meson/pull/12532
+    patch("configure_file-1.3.0.patch", when="@1.3.0")
```

Comment:
Can we make this a URL patch for https://github.com/mesonbuild/meson/commit/9016e6958bb83feb9a724f20d8badb116bf7c5f2.patch?full_index=1 instead to avoid having another patch in the repository?

Reply:
Yes we can.